src/main/java/com/github/crimsondawn45/fabricshieldlib/initializers/FabricShieldLib.java

```diff
@@ -92,13 +103,31 @@ public void onInitialize() {
             logger.warn("FABRIC SHIELD LIB DEVELOPMENT CODE RAN!!!, if you are not in a development environment this is very bad! Test items and test enchantments will be ingame!");
 
             //Register Custom Shield
-            fabric_banner_shield = registerItem("fabric_banner_shield", (props) -> new FabricBannerShieldItem(props.maxDamage(336).component(MODEL_COMPONENT, new FabricShieldModelComponent(FabricShieldLibClient.FABRIC_BANNER_SHIELD_BASE.getTextureId(), FabricShieldLibClient.FABRIC_BANNER_SHIELD_BASE_NO_PATTERN.getTextureId(), FabricShieldLibClient.fabric_banner_shield_model_layer.toString())), 85, 9, Items.OAK_PLANKS, Items.SPRUCE_PLANKS));
+            fabric_banner_shield = registerItem("fabric_banner_shield",
+                    (props) -> new FabricBannerShieldItem(props.maxDamage(336)
```

Comment:
Use the constant defined in `FabricShieldUtils` rather than hardcoding it:
336 -> `FabricShieldUtils.VANILLA_SHIELD_DURABILITY`